backend/controllers/rate.js

```diff
@@ -0,0 +1,22 @@
+// Load required packages
+var Rate = require('../models/history.js');
+
+// Create endpoint /api/ratings for POST
+exports.postRates = function (req, res) {
+  var movieId = req.body.movieId;
+  var score = req.body.score;
+  var userId = req.user.id;
+  // Save the rating and check for errors
+  Rate.build({score: score, movie_id: movieId, user_id: userId})
+      .save().then(function (success) {
+        res.json({message: 'Ratings successfully posted!'});
```

Comment:
line 12: JSON format, I think break lines around curly brackets should be used, even though it is just one line son